### ISSUE TITLE
Depth 3 feature always equals to 0

### ISSUE THREAD

**Issue by Ingvar-Y** [Featuretools/featuretools#643]
### Bug/Feature Request Title
When I try to create depth 3 seed feature it always returns a column equal to 0 instead of expected values.
#### Bug/Feature Request Description
I built an example dataset described like this:
```python
Entityset: None
  Entities:
    pers [Rows: 1, Columns: 4]
    cli [Rows: 29, Columns: 5]
    k [Rows: 43, Columns: 4]
    fin_exp [Rows: 43, Columns: 4]
  Relationships:
    cli.hid -> pers.hid
    k.id_k -> cli.id_k
    fin_exp.id_fin -> k.id_fin
```
When I try to calculate a seed feature of depth 3 like
```python
ft.Feature(ft.Feature(ft.Feature(tset['fin_exp']['premium'], parent_entity = tset['k'], primitive = Sum()), parent_entity = tset['cli'], primitive = Sum()), parent_entity = tset['pers'], primitive = Sum())
ft.Feature(ft.Feature(ft.Feature(tset['fin_exp']['id_fin'], parent_entity = tset['k'], primitive = NumUnique()), parent_entity = tset['cli'], primitive = Sum()), parent_entity = tset['pers'], primitive = Sum())
```
it returns a column (one value in this case) fulls of zeroes instead of actual values. For comparison if I count on a duplicate `tset['k']['idf']` of index `tset['k']['id_fin']` for the parent entity
```python
ft.Feature(ft.Feature(tset['k']['idf'], parent_entity = tset['cli'], primitive = NumUnique()), parent_entity = tset['pers'], primitive = Sum())
```
I get 43 - as expected since fin_exp and k have one-to-one relationship in the example.
#### Expected Output
```python
ft.Feature(ft.Feature(ft.Feature(tset['fin_exp']['id_fin'], parent_entity = tset['k'], primitive = NumUnique()), parent_entity = tset['cli'], primitive = Sum()), parent_entity = tset['pers'], primitive = Sum())
ft.Feature(ft.Feature(tset['k']['idf'], parent_entity = tset['cli'], primitive = NumUnique()), parent_entity = tset['pers'], primitive = Sum())
```
should return the same value.
#### Output of ``featuretools.show_info()``
featuretools.show_info() still doesn't print any input. Branch is slightly behind master.
<details>
Featuretools version: 0.9.0
Featuretools installation directory: y:\git\featuretools\featuretools

SYSTEM INFO
-----------
python: 3.7.3.final.0
python-bits: 64
OS: Windows
OS-release: 2008ServerR2
machine: AMD64
processor: Intel64 Family 6 Model 45 Stepping 2, GenuineIntel
byteorder: little
LC_ALL: None
LANG: None
LOCALE: None.None

INSTALLED VERSIONS
------------------
numpy: 1.16.4
pandas: 0.24.2
tqdm: 4.32.2
toolz: 0.9.0
PyYAML: 5.1.1
cloudpickle: 1.2.1
future: 0.17.1
dask: 2.0.0
distributed: 2.0.1
psutil: 5.6.3
Click: 7.0
scikit-learn: 0.21.2
pip: 19.1.1
setuptools: 41.0.1

</details>

**Comment by kmax12** [comment]
@Ingvar-Y can you share a sample of your data so we can try to recreate the issue as you describe? 

also, please upgrade to featuretools `0.9.1` and uninstall/reinstall. if `featuretools.show_info()` isn't working something else may be wrong

**Comment by Ingvar-Y** [comment]
Ok, I understand what happens with `featuretools.show_info()`. It prints results in terminal but doesn't show them in Jupyter notebook. That isn't convenient but it works.

I'll attach an example with data sample a bit later.

**Comment by kmax12** [comment]
@Ingvar-Y ahh, thanks for pointing out that issue about `ft.show_info()` not working in juypter notebook. do you think you can open a new issue on that, so we can fix?

We will keep our eye out for the sample data.

Thanks!

**Comment by Ingvar-Y** [comment]
[example.zip](https://github.com/Featuretools/featuretools/files/3369934/example.zip)
I hope you don't mind parquet files since they allow to preserve column types.
A reproducible example:
```python
import featuretools as ft
import pandas as pd
from featuretools.primitives import NumUnique, Sum
tpers = pd.read_parquet('tpers.parquet')
tcli = pd.read_parquet('tcli.parquet')
tk = pd.read_parquet('tk.parquet')
tfin = pd.read_parquet('tfin.parquet')
tset = ft.EntitySet()
tset = tset.entity_from_dataframe(entity_id = 'pers',
                                  dataframe = tpers,
                                  index = 'hid',
                                  variable_types = {'birthdate': ft.variable_types.DateOfBirth,
                                                    'gender': ft.variable_types.Categorical,
                                                    'corporate_client': ft.variable_types.Categorical})
tset = tset.entity_from_dataframe(entity_id = 'cli',
                                  dataframe = tcli,
                                  index = 'id_k',
                                  variable_types = {'hid': ft.variable_types.Id,
                                                    'group': ft.variable_types.Categorical,
                                                    'activation': ft.variable_types.Categorical},
                                  time_index = 'startdate')
tset = tset.entity_from_dataframe(entity_id = 'k',
                                  dataframe = tk,
                                  index = 'id_fin',
                                  variable_types = {'id_k': ft.variable_types.Id,
                                                    'enddate': ft.variable_types.Datetime},
                                  time_index = 'startdate')
tset = tset.entity_from_dataframe(entity_id = 'fin_exp',
                                  dataframe = tfin,
                                  index = 'id_temp',
                                  variable_types = {'id_fin': ft.variable_types.Id,
                                                    'premium': ft.variable_types.Numeric},
                                  make_index = True,
                                  time_index = 'premiumdate')
tset = tset.add_relationship(ft.Relationship(tset['pers']['hid'], tset['cli']['hid']))
tset = tset.add_relationship(ft.Relationship(tset['k']['id_fin'], tset['fin_exp']['id_fin']))
tset = tset.add_relationship(ft.Relationship(tset['cli']['id_k'], tset['k']['id_k']))
tmat, tdef = ft.dfs(entityset = tset,
                    target_entity = 'pers',
                    agg_primitives = [],
                    trans_primitives = [],
                    seed_features = [ft.Feature(ft.Feature(ft.Feature(tset['fin_exp']['premium'], parent_entity = tset['k'],
                                                                      primitive = Sum()),
                                                           parent_entity = tset['cli'], primitive = Sum()),
                                                parent_entity = tset['pers'], primitive = Sum()),
                                     ft.Feature(ft.Feature(ft.Feature(tset['fin_exp']['id_fin'], parent_entity = tset['k'],
                                                                      primitive = NumUnique()),
                                                           parent_entity = tset['cli'], primitive = Sum()),
                                                parent_entity = tset['pers'], primitive = Sum()),
                                     ft.Feature(ft.Feature(tset['k']['idf'], parent_entity = tset['cli'],
                                                           primitive = NumUnique()),
                                                parent_entity = tset['pers'], primitive = Sum())
                                    ])
```

**Comment by kmax12** [comment]
thank you @Ingvar-Y for reporting this issue. it has been fixed in #668.

for the time being, you can install the `set-index-featureset-calculator` branch to get the fix.

```
pip install -e git://github.com/featuretools/featuretools.git@set-index-featureset-calculator#egg=featuretools
```

The fix will go out in the next featuretools release.

thanks again

**Comment by Ingvar-Y** [comment]
@kmax12 I can confirm that the issue is fixed in `set-index-featureset-calculator` branch.
